### PR TITLE
updated to jekyll 3.9.2 partially resolving issue  (#313)

### PR DESCRIPTION
**Pull request by nick-mon1**
WIP, currently blocked at `jekyll 3.9.2`.

Resolves issue #313 

- [ ]  Update to Jekyll 4.2.2 
- [x] Use ruby >= 2.4.0
- [x] Update bundler
- [x]  Update bundler dependencies

Jekyll is blocked with the following gem dependencies:

> Bundler could not find compatible versions for gem "jekyll":
>   In Gemfile:
>     jekyll (~> 4.2.2)
> 
>     jekyll_pages_api_search was resolved to 0.5.0, which depends on
>       jekyll_pages_api (~> 0.1.4) was resolved to 0.1.6, which depends on
>         jekyll (>= 2.0, < 4.0)

May need to consider replacing `jekyll_pages_api` with another gem.

- [https://github.com/18F/jekyll_pages_api/issues/47](https://github.com/18F/jekyll_pages_api/issues/47)
- [https://github.com/18F/jekyll_pages_api_search](https://github.com/18F/jekyll_pages_api_search)

### CODE REVIEW

**Comment by nick-mon1** [comment]
Implementing [search.gov](https://search.gov/get-started/) search service will require some additional effort, this would be a multi-step process and some coordination with the search.gov team to get search integrated with the site. Additional testing will be required to make sure the search results page looks good and performs as expected.

In the meantime we can update to `jekyll 3.9.2` and still use the current search features until we decide to complete the search.gov integration.

- [Introduction to Search.gov](https://www.youtube.com/watch?v=niT9kU6AfP8)

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "jekyll", "~> 3.8.5"
+# gem "jekyll", "~> 4.2.2"
```
Comment (mejiaj) [inline Gemfile:3]: Shouldn't this be `gem "jekyll", "~> 3.9.2"`?

**Review by mejiaj — CHANGES_REQUESTED** [review]
Looks good so far, just a few things. A question about commented line regarding jekyll version. 

Could we also look into updating `.bundler-version`? The last update was 3 years ago.

File: Gemfile (hunk 1 above)
Comment (nick-mon1) [reply Gemfile]: Yes, this has been corrected!

**Comment by nick-mon1** [comment]
I've updated to `bundler 2.3.15`, the most current version.

**Pull request merged**